daffodil-cli/src/main/scala/org/apache/daffodil/Main.scala

```diff
@@ -873,8 +883,6 @@ object Main {
 
               val parseResult = eitherOutputterOrHandler match {
                 case Right(saxContentHandler) =>
-                  // reset in case we are streaming
-                  saxContentHandler.reset()
```

Comment:
Why is this reset removed? Seems like this is important?

Reply:
Init protocol was converted over putting it on startDocument/endDocument which is where SAX APIs want you to do it, so that you don't have to extend SAX handler classes with additional methods.